src/commands/mod.rs

```diff
@@ -1437,10 +1439,10 @@ fn log_template(settings: &UserSettings) -> String {
     };
     let default_template = format!(
         r#"
-            label(if(current_working_copy, "working_copy"),
```

Comment:
I'm wondering if we could make this less of a mess if we had immutable variables and pure functions in the templating language.

Hopefully, things like 

```
if(divergent,
                  label("divergent", change_id.{prefix_format} "??"),
                  label("divergent", "{{change_id.{prefix_format}}}??"),
```

could then be defined outside this expression as variables, with less escaping.

 It seems like it should be possible, but not without some work on the design.

Reply:
I would also replace string interpolation with a `format` function if we go this way. Anything to make things more modular.

Reply:
I think this can also be addressed by template aliases. Aliases wouldn't be local to this template, but I think we can add some scoped aliases + template set if needed.

Reply:
That seems reasonable to me. I don't see that locality of variables would be particularly important or useful here. There's no reason template aliases can't be functions if that's helpful at some point.